nipype/pipeline/plugins/base.py

```diff
@@ -543,7 +543,7 @@ def _submit_job(self, node, updatehash=False):
         pyscript = create_pyscript(node, updatehash=updatehash)
         batch_dir, name = os.path.split(pyscript)
         name = ".".join(name.split(".")[:-1])
-        batchscript = "\n".join((self._template, "%s %s" % (sys.executable, pyscript)))
+        batchscript = "\n".join((self._template[:-1], "%s %s" % (sys.executable, pyscript)))
```

Comment:
Looking at this, I think we want to be a little more discriminating, as it's not guaranteed that a template file will end in a newline. What if we did :

```suggestion
        batchscript = "\n".join(
            (self._template.rstrip("\n"), "%s %s" % (sys.executable, pyscript))
        )
```

Reply:
Indeed, my approach was naïve.